src/solvers/dgsem_t8code/containers_2d.jl

```diff
@@ -4,6 +4,7 @@
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
 #! format: noindent
+#! format: off
```

Comment:
?

Reply:
You removed `#! format: noindent`, the line `#! format: off` is still there?

Reply:
@ranocha I brought the file back to its original state.